### DISCUSSION TITLE
[SOLVED] No puedo editar usuarios en detalle de tarea

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#109] [4. Bug Reports] [answered]
Hola

No sé si está bien el nombre del título pero quiero saber lo siguiente:

Cuando entro a un canal de task de tipo bound, veo la información de la task (presionando en el número de la task): ¿Yo puedo modificar alguna información dentro de ésta? (ejemplo: Sacar usuarios del canal, agregar información de los campos adicionales, cambiar información de la tarea.

La duda la tengo, porque actualmente sólo puedo hacerlo a través de la vista de tareas y no en la vista del canal.

Como referencia a lo que indico adjunto SS marcando en color rojo lo que estoy diciendo:

![image](https://user-images.githubusercontent.com/61419471/116151658-e1f12980-a6b2-11eb-8842-f35ef7b38fa4.png)

Gracias
:)

**Comment by acedward** [comment]
Si eres un editor, deberias poder modificar todos los campos

**Reply by NicolasPLG** [reply]
@acedward entonces no se encuentra habilitado esa opción :(. O es un bug o es una feature ?

**Reply by acedward** [reply]
Si eres un editor de la tarea, puedes editar todo.
El assignado tiene capacidad limitada, puede realizar solo algunas acciones.
Si no eres editor ni asignado, no puedes agregar más personas a la tarea - ni editar campos.

Si no funciona así, por favor sube un video por aquí, porque lo probé y me funciona correctamente.

**Reply by NicolasPLG** [reply]
te dejo el link @acedward porque pesa muhco xd https://drive.google.com/file/d/1x1no7eOFuFAN-MQY3Ijr2QdU_wtjxS01/view?usp=sharing

Luego si está bien ese video lotrataré de comprimir hata 10 mb

**Reply by acedward** [reply]
Yo no veo ese canal siendo que estoy en la lista.  
Podría ser que esta una tarea muy antigua y está desincronizado el canal y la tarea (eliminado usuarios manualmente del canal)

**Reply by NicolasPLG** [reply]
Entiendo, pero sería temas de caché o un bug de visualización al que edita el canal?.
O si es una desincronización del canal, cómo se puede solucionar?

**Comment by rafaelwalter** [comment, marked as answer]
@NicolasPLG es un bug :(

Creamos un ticket interno con numero 2786.

Gracias por el reporte!!!!
